metadatastore/commands.py

```diff
@@ -377,13 +380,61 @@ def _format_time(search_dict):
 
 
 def _normalize_human_friendly_time(val):
-    "Parse '2015', '2015-03', '2015-03-30', and '2015-03-30 18:00:00'."
+    """Given a string ('2015', '2015-03', '2015-03-30', and
+    '2015-03-30 18:00:00'), datetime (datetime.datetime.now()), with
+    or without tzinfo), or timestamp (time.time()), return a timestamp
+    (seconds since jan 1 1970 UTC).  Non string/datetime.datetime
+    values are returned unaltered.
+    """
+
     tz = conf.connection_config['timezone']  # e.g., 'US/Eastern'
-    if isinstance(val, six.string_types) or isinstance(val, datetime.datetime):
-        return pd.to_datetime(val).tz_localize(tz).value/1e9
-    else:
+    zone = pytz.timezone(tz)  # tz as datetime.tzinfo object
+    epoch = pytz.UTC.localize(datetime.datetime(1970, 1, 1))
+    check = True
+
+    if isinstance(val, six.string_types):
+        # unix 'date' cmd format '%a %b %d %H:%M:%S %Z %Y' works but
+        # doesn't get TZ?
+
+        formats = ['%Y-%m-%d %H:%M:%S',
+                   '%Y-%m-%d %H:%M',  # these 2 are not as doc'd, but
```

Comment:
Can you update the docstring to match all of the allowed types?  One trick we can use to keep the code and docstring in sync is to put the format list out side as 

```python
FORMATS = ['a' , 'b', 'c']

_doc_formats = '\n'.join('\t- {}'.format(_) for _ in FORMATS)

def test_func_format():
    """
    This is a test, the allowed strings are

    {}
    """
    pass

test_func_format.__doc__ = test_func_format.__doc__.format(_doc_formats)
```